src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -589,6 +595,7 @@ class SwitchHelper {
      */
     List<FloodlightResourceAddress> knockoutSwitch(Switch sw, FloodlightConnectMode mode, boolean waitForRelatedLinks) {
         def blockData = lockKeeper.knockoutSwitch(sw, mode)
+        cleanupManager.addAction(REVIVE_SWITCH, {reviveSwitch(sw, blockData, true)}, CleanupAfter.TEST)
```

Comment:
This cleanup action is added after the switch knockout and can be a potential place of failure during recovery. I mean we can get an error during knockout and our cleanup action REVIVE_SWITCH will not be a part of the cleanup. WDYT to collect this data from floodlight?

Reply:
This could be a little bit tricky. Let's do it in the next parts of refactoring.